lua/neogit/popups/tag/init.lua

```diff
@@ -0,0 +1,27 @@
+local actions = require("neogit.popups.tag.actions")
+local popup = require("neogit.lib.popup")
+
+local M = {}
+
+function M.create()
+  local p = popup
+    .builder()
+    :name("NeogitTagPopup")
+    :arg_heading("Arguments")
+    :switch("f", "force", "Force")
+    :switch("a", "annotate", "Annotate")
+    :switch("s", "sign", "Sign")
+    :switch("u", "local-user", "Sign as", { enabled = false })
+    :new_action_group("Create")
```

Comment:
The first action group is implicit, so you don't need to make a new one. You can just use:
```suggestion
    :group_heading("Create")
```